<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QMainWindow" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>main</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="graphic">
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>623</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="separator"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="separator"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuComponents">
    <property name="title">
     <string>Components</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionEditor"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuComponents"/>
   <addaction name="menuSetting"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionRun"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionzoomPlus"/>
   <addaction name="actionzoomLess"/>
   <addaction name="actionadjust"/>
   <addaction name="separator"/>
   <addaction name="actionChange"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar_editor">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSendCode"/>
   <addaction name="separator"/>
   <addaction name="actionChange"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normalon>:/Resource/new.png</normalon>
     <selectedon>:/Resource/new.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/open.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/save.jpeg</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/saveas.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/exit.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/undo.jpeg</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/redo.jpeg</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/cut.jpeg</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/copy.jpeg</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/paste.jpeg</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/delete.jpeg</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionLed">
   <property name="text">
    <string>Led</string>
   </property>
  </action>
  <action name="actionButton">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Button</string>
   </property>
   <property name="iconText">
    <string notr="true">Button</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/run.jpeg</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/stop.jpeg</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="iconText">
    <string>Stop</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionChange">
   <property name="text">
    <string>Editor</string>
   </property>
  </action>
  <action name="actionEditor">
   <property name="text">
    <string>Editor</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionzoomPlus">
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/+zoom.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>zoom</string>
   </property>
  </action>
  <action name="actionzoomLess">
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/-zoom.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>zoom</string>
   </property>
  </action>
  <action name="actionadjust">
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/adjust.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>adjust</string>
   </property>
  </action>
  <action name="actionSendCode">
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/aceptar.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>SendCode</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
